api/app/signals/apps/api/serializers/signal_context.py

```diff
@@ -74,7 +80,20 @@ class Meta:
         )
 
     def get_geography(self, obj):
-        return None
+        signals_for_geography_qs = Signal.objects.annotate(
+            distance_from_point=Distance('location__geometrie', obj.location.geometrie),
```

Comment:
This solution is tested on a large set of data, because I also was wondering what would happen, and I did not encountered any problems.

@tcoenen perhaps this is something that you can take a look at. I think you have a bit more knowledge about this and maybe can come up with a better solution. And hopefully explain it to me so that I also become a bit more familiar with this.

Reply:
This was discussed, we added a number of backlog items to tackle geometry versus geography data types.